source/agg-pipeline-builder/pipeline-custom-collation.txt

```diff
@@ -0,0 +1,54 @@
+.. _pipeline-custom-collation:
+
+========================
+Specify Custom Collation
+========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Use custom collation to specify language-specific rules for string comparison, 
+such as rules for letter case and accent marks.
```

Comment:
[suggestion]

For users arriving through Google, it isn't clear that we're talking about specifying collation for a pipeline. Can we mention that bit of context somewhere in the introduction?

Reply:
Will do! Do you think it'd also be helpful to mention that it's for a pipeline in the title? I noticed that the rest of the pages so far mention "pipeline," so I thought it could be helpful to edit that.